src/components/Floating/FloatingButton.tsx

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+import Image from 'next/image';
+import { twMerge } from 'tailwind-merge';
+
+type FloatingButtonProps = {
+  type: 'top' | 'ask';
+  onClick?: () => void;
+};
+
+export function FloatingButton({ type, onClick }: FloatingButtonProps) {
+  const handleClick = useCallback(() => {
+    if (type === 'top') {
+      window.scrollTo({ top: 0, behavior: 'smooth' });
+    } else if (type === 'ask' && onClick) {
+      onClick();
+    }
+  }, [type, onClick]);
```

Comment:
```suggestion
  const handleClick = () => {
    if (type === 'top') {
      window.scrollTo({ top: 0, behavior: 'smooth' });
    } else if (type === 'ask' && onClick) {
      onClick();
    }
  };

```
P5: useCallback은 여기서 꼭 사용해주지 않아도 될 것 같다는 생각이 들어요..!